pkg/k8sutil/label.go

```diff
@@ -0,0 +1,13 @@
+package k8sutil
+
+import "fmt"
+
+// NewPrefixedName returns a name prefixed by prefix and with length that is no longer than 63
+// chars
+func NewPrefixedName(prefix, original string) string {
```

Comment:
Would you mind using this in Migrate? (specifically in `newPvcName`)

(This might require changing this to be `NewSuffixedName` for consistency)

Reply:
will take this suggestion as a future TODO.